src/tetris/scene/playScene.ts

```diff
@@ -11,8 +11,65 @@ import Vector2 = Phaser.Math.Vector2;
 import NetworkingClient from "tetris/networking/networkingClient";
 import FieldState from "tetris/field/fieldState";
 import RemoteField from "tetris/field/remoteField";
-
-const PLAYER_FIELD_DRAW_OFFSET: Vector2 = new Vector2(50, 80);
+import CountdownWidget from "tetris/ui/countdownWidget";
+import ScoreboardWidget from "tetris/ui/scoreboardWidget";
+import ScoreWidget from "tetris/ui/scoreWidget";
+
+const PLAYER_FIELD_DRAW_OFFSET: Vector2 = new Vector2(
+	(config.graphics.width - config.field.width * config.field.blockSize) / 2, 
+	(config.graphics.height - config.field.height * config.field.blockSize) / 1.5);
+
+const players = [
```

Comment:
Shouldn't we remove this static array? Or is it for illustrating purpose and gets removed with a future PR?

Reply:
Yes, it's just for illustration purposes and will be removed later.